addons/l10n_ch/report/swissqr_report.xml

```diff
@@ -31,7 +31,7 @@
                     <h1>QR-bill for invoice <t t-esc="o.name"/></h1>
                 </div>
 
-                <div class="swissqr_content">
+                <div class="swissqr_content" t-attf-style="position: relative; top: -#{report_paperformat_id.margin_top * 1.27}mm">
```

Comment:
Could you explain why this value `1.27` ?
A comment would be helpful.

Reply:
Everything is zoomed to 1.25 (and it's 1.27 as value because there must be a rounding error depending on margin size).

But this PR will probably not be merged and instead allowing to have padding on the header:

https://github.com/odoo/odoo/pull/54383/files